clink/activate_clink.bat

```diff
@@ -0,0 +1,2 @@
+@echo off
+call clink inject -q --profile "%LOCALAPPDATA%"\clink

```

Comment:
Is there a way to make sure that clink is removed cleanly when this package is uninstalled?